bigbluebutton-html5/imports/api/users/server/modifiers/setConnectionStatus.js

```diff
@@ -0,0 +1,40 @@
+import { Meteor } from 'meteor/meteor';
+import { check } from 'meteor/check';
+import Users from '/imports/api/users';
+import Logger from '/imports/startup/server/logger';
+
+const VALID_CONNECTION_STATUS = ['online', 'offline'];
+
+export default function setConnectionStatus(meetingId, userId, status = 'online') {
+  check(meetingId, String);
+  check(userId, String);
+  check(status, String);
+
+  if (!VALID_CONNECTION_STATUS.includes(status)) {
+    throw new Meteor.Error('invalid-connection-status',
+      `Invalid connection status, received ${status} expecting ${VALID_CONNECTION_STATUS.join()}`);
+  }
+
+  const selector = {
+    meetingId,
+    userId,
+  };
+
+  const modifier = {
+    $set: {
+      'user.connection_status': status,
+    },
+  };
+
+  const cb = (err, numChanged) => {
+    if (err) {
+      return Logger.error(`Updating connection status user=${userId}: ${err}`);
+    }
+
+    if (numChanged) {
```

Comment:
When refreshing the html5 client, its user disappears from the meeting indefinitely (from Flash's perspective). The html5 client user is still able to send chat however